tests/setup.sh

```diff
@@ -0,0 +1,30 @@
+#!/usr/bin/env bash -x
+source "$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)/test_helper.bash"
+
+BIN_STUBS="$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)/bin"
+mkdir -p $BIN_STUBS
+
+if [[ ! -d $DOKKU_ROOT ]]; then
+  git clone https://github.com/progrium/dokku.git $DOKKU_ROOT > /dev/null
+fi
+
+cd $DOKKU_ROOT
+echo "Dokku version $DOKKU_VERSION"
+git checkout $DOKKU_VERSION > /dev/null
+cd -
+
+# TODO: fix or remove before merge
+#source "$(dirname "$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)")/config"
```

Comment:
Thanks! I'm usually pretty good about not letting these things make it to the PR stage 😸